libass/ass_types.h

```diff
@@ -98,32 +98,32 @@ typedef struct ass_library ASS_Library;
 
 /* ASS Style: line */
 typedef struct ass_style {
-    char *Name;
-    char *FontName;
+    char *Name;     //must be a valid non-NULL string pointer
+    char *FontName; //must be a valid non-NULL string pointer
```

Comment:
Specify how these should be allocated? afaik they have to come from the same copy of `malloc` as libass uses (which has some implications on windows)

Also, worth noting that while they can't be null, they can point to empty strings.